app/controllers/concerns/authorization.rb

```diff
@@ -2,26 +2,25 @@ module Authorization
   extend ActiveSupport::Concern
 
   included do
+    helper_method :unauthorized!
     helper_method :authorize_super_admin!
     helper_method :authorize_admin!
-    helper_method :authorize_deployer!
-    helper_method :unauthorized!
-  end
-
-  def authorize_super_admin!
-    unauthorized! unless current_user.is_super_admin?
   end
 
-  def authorize_admin!
-    unauthorized! unless current_user.is_admin?
+  def unauthorized!
+    # Eventually to UnauthorizedController
+    throw(:warden)
   end
 
   def authorize_deployer!
     unauthorized! unless current_user.is_deployer?
   end
 
-  def unauthorized!
-    # Eventually to UnauthorizedController
-    throw(:warden)
+  def authorize_super_admin!
+    unauthorized! unless current_user.is_super_admin?
+  end
+
+  def authorize_admin!
+    unauthorized! unless current_user.is_admin?
```

Comment:
Yeah... this was a result of some initial hesitation around adding here the project level authorisation helper methods or creating a new module.

Answering your question, I just removed the authorise_deployer helper method (no longer needed).

I'm going to revert the file form master and apply the same changes again. Easy enough.